app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainViewModel.kt

```diff
@@ -23,63 +23,65 @@ class MainViewModel : ViewModel() {
         globalData()
     }
 
-    fun fetchGlobalData() {
+    private fun fetchGlobalData() {
         /**
          * all cases service to fetch global covid19 data
          */
         globalData = casesService.fetchGlobalData()
     }
 
-    fun globalData() {
+    private fun globalData() {
         GlobalScope.launch(context = Dispatchers.Main) {
-            var gdp : GlobalDataProvider = GlobalDataProvider()
+            val gdp = GlobalDataProvider()
 
             /**
              * Call API to retrieve specific global covid19 data
              */
-            var gCases: ArrayList<String>? = gdp.getGlobalCovidData("total_cases")
-            var gActive: ArrayList<String>? = gdp.getGlobalCovidData("active_cases")
-            var gTotalDeath: ArrayList<String>? = gdp.getGlobalCovidData("total_deaths")
-            var gRecovered: ArrayList<String>? = gdp.getGlobalCovidData("total_recovered")
-            var gNewCases: ArrayList<String>? = gdp.getGlobalCovidData("new_cases")
-            var gNewDeaths: ArrayList<String>? = gdp.getGlobalCovidData("new_deaths")
-            var gSeriousCritical: ArrayList<String>? = gdp.getGlobalCovidData("serious_critical")
-            var gCasesPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("total_cases_per_1m_population")
-            var gDeathPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("deaths_per_1m_population")
-            var gStatistics: ArrayList<String>? = gdp.getGlobalCovidData("statistic_taken_at")
+            val gCases: ArrayList<String>? = gdp.getGlobalCovidData("total_cases")
+            val gActive: ArrayList<String>? = gdp.getGlobalCovidData("active_cases")
+            val gTotalDeath: ArrayList<String>? = gdp.getGlobalCovidData("total_deaths")
+            val gRecovered: ArrayList<String>? = gdp.getGlobalCovidData("total_recovered")
+            val gNewCases: ArrayList<String>? = gdp.getGlobalCovidData("new_cases")
+            val gNewDeaths: ArrayList<String>? = gdp.getGlobalCovidData("new_deaths")
+            val gSeriousCritical: ArrayList<String>? = gdp.getGlobalCovidData("serious_critical")
+            val gCasesPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("total_cases_per_1m_population")
+            val gDeathPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("deaths_per_1m_population")
+            val gStatistics: ArrayList<String>? = gdp.getGlobalCovidData("statistic_taken_at")
```

Comment:
This comment will be used for all changes of var to val, I believe these are constant variables: https://kotlinlang.org/docs/tutorials/kotlin-for-py/declaring-variables.html